### DISCUSSION TITLE
Schema translation using MM2

### DISCUSSION THREAD

**Discussion by Sharad-SKJ052** [strimzi/strimzi-kafka-operator#8612] [Q&A]
Hi Team ,

We are using strimzi mm2 version 3.0.0 to replicate data between two Kafka clusters which has two different schema registries.
While consuming messages on destination Kafka cluster the consumer application is referring to the source schema id and its failing to consume. 
Is there any way so that our consumer should refer to schema available in the local schema registry.
We came across this MessageHandler to solve this issue but how do we use that in Strimzi MM2 .
Any guideline would be useful

**Comment by scholzj** [comment]
> We came across this MessageHandler to solve this issue but how do we use that in Strimzi MM2 .

Explaining what message handler you came across would probably be a good start. Then it might be easier to understand if and how you can use that with Strimzi.

**Reply by Sharad-SKJ052** [reply]
We checked the below link regarding MessageHandler but it is not mentioning Strimzi 
https://stackoverflow.com/questions/56671069/how-to-replicate-schema-with-kafka-mirror-maker

**Reply by scholzj** [reply]
Well, that is several years old and IMHO applies only to Mirror Maker 1.

**Reply by scholzj** [reply]
But as explained there -> Apache Kafka knows nothing about how Confluent Schema Registry works because the registry is not part of the Apache Kafka project, so it will not do any translation for you. That applies to all Kafka components I think.
